src/Cake.Common/Tools/ReportGenerator/ReportGeneratorSettings.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.ReportGenerator
+{
+    /// <summary>
+    /// Contains settings used by <see cref="ReportGeneratorRunner"/>.
+    /// </summary>
+    [CakeAliasCategory("ReportGenerator")]
+    public sealed class ReportGeneratorSettings
```

Comment:
Yes, that's what I've done (and more). Problem is that this will break
addins that are not compiled against that version.
On Oct 23, 2015 18:02, "Yves Schmid" <notifications@github.com> wrote:

> In src/Cake.Common/Tools/ReportGenerator/ReportGeneratorSettings.cs
> <https://github.com/cake-build/cake/pull/467#discussion_r42882426>:
>
> > @@ -0,0 +1,48 @@
> > +﻿using Cake.Core.Annotations;
> > +using Cake.Core.IO;
> > +
> > +namespace Cake.Common.Tools.ReportGenerator
> > +{
> > +    /// <summary>
> > +    /// Contains settings used by <see cref="ReportGeneratorRunner"/>.
> > +    /// </summary>
> > +    [CakeAliasCategory("ReportGenerator")]
> > +    public sealed class ReportGeneratorSettings
>
> @gep13 <https://github.com/gep13> probably because of something like this:
>
> public abstract class Tool<TSettings> where TSettings:BaseSettings
> {
> ...
> }
>
> which would get rid of all the toolPath parameters in the Runmethods.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cake-build/cake/pull/467/files#r42882426>.
>